spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitGatewaySupport.java

```diff
@@ -32,6 +32,7 @@
  * through overriding the {@link #createRabbitTemplate} method.
  *
  * @author Mark Pollack
+ * @author Gary Russell
  * @see #setConnectionFactory
  * @see #setRabbitTemplate
```

Comment:
There is no such a method any more...
```
D:\SpringIO\spring-amqp\spring-rabbit\src\main\java\org\springframework\amqp\rabbit\core\RabbitGatewaySupport.java:37: error: reference not found
 * @see #setRabbitTemplate
        ^
```
I wonder when you will switch to IDEA to simplify the life :smile: : 
![no_method](https://cloud.githubusercontent.com/assets/1071590/19325197/f943033c-9092-11e6-9b64-e3ec3fcc4c63.png)

Will fix on merge.